applications/StructuralMechanicsApplication/custom_elements/small_displacement_mixed_volumetric_strain_element.cpp

```diff
@@ -1143,32 +1597,134 @@ void SmallDisplacementMixedVolumetricStrainElement::CalculateOnIntegrationPoints
+
+            // Set the constitutive variables
+            CalculateConstitutiveVariables(
+                kinematic_variables,
+                constitutive_variables,
+                cons_law_values,
+                i_gauss,r_integration_points,
+                ConstitutiveLaw::StressMeasure_Cauchy);
+
+            // Calculate tau_1 stabilization constant
+            const double tau_1 = CalculateTau1(m_T, kinematic_variables, constitutive_variables, rCurrentProcessInfo);
+
+            // Get current Gauss point data
+            const double bulk_modulus = CalculateBulkModulus(constitutive_variables.D);
+            noalias(body_force) = GetBodyForce(r_geometry.IntegrationPoints(GetIntegrationMethod()), i_gauss);
+            noalias(grad_eps) = prod(trans(kinematic_variables.DN_DX), kinematic_variables.VolumetricNodalStrains);
+
+            // If dynamic subscales are active, add the dynamic subscale component
+            aux_u_s = bulk_modulus * grad_eps;
+            for (IndexType d = 0; d < dim; ++d) {
+                aux_u_s[d] += body_force[d];
+            }
+            if (mIsDynamic) {
+                aux_u_s += (density / std::pow(dt,2)) * (2.0*mDisplacementSubscale1[i_gauss] - mDisplacementSubscale2[i_gauss]);
+            }
+
+            // Multiply by the stabilization parameter
+            aux_u_s *= tau_1;
+
+            // Save current value in the output container
+            for (IndexType d = 0; d < dim; ++d) {
+                rOutput[i_gauss][d] = aux_u_s[d];
+            }
+        }
+    } else {
+        CalculateOnConstitutiveLaw(rVariable, rOutput, rCurrentProcessInfo);
+    }
+}
+
+/***********************************************************************************/
+/***********************************************************************************/
+
 const Parameters SmallDisplacementMixedVolumetricStrainElement::GetSpecifications() const
 {
     const Parameters specifications = Parameters(R"({
-        "time_integration"           : ["static"],
+        "time_integration"           : ["static","dynamic"],
         "framework"                  : "lagrangian",
         "symmetric_lhs"              : true,
         "positive_definite_lhs"      : true,
         "output"                     : {
             "gauss_point"            : ["CAUCHY_STRESS_VECTOR"],
             "nodal_historical"       : ["DISPLACEMENT","VOLUMETRIC_STRAIN"],
             "nodal_non_historical"   : [],
-            "entity"                 : []
+            "entity"                 : ["DISPLACEMENT_SUBSCALE", "VOLUMETRIC_STRAIN_SUBSCALE"]
         },
         "required_variables"         : ["DISPLACEMENT","VOLUMETRIC_STRAIN"],
         "required_dofs"              : [],
         "flags_used"                 : [],
         "compatible_geometries"      : ["Triangle2D3", "Quadrilateral2D4", "Tetrahedra3D4","Hexahedra3D8"],
-        "element_integrates_in_time" : true,
+        "element_integrates_in_time" : false,
```

Comment:
It is not true now and false then?

Reply:
This means that the time integration is done outside the element (via scheme) so `false` is correct.